core/config/v2/docs/secrets.toml

```diff
@@ -52,4 +52,8 @@ Username = "A-Mercury-Username" # Example
 # Password is used for basic auth of the Mercury endpoint
 Password = "A-Mercury-Password" # Example
 # URL is the Mercury endpoint URL which is used by OCR2 Automation to access Mercury price feed
-URL = "https://mercury.stage.link" # Example
\ No newline at end of file
+URL = "https://mercury.stage.link" # Example
+
+[Threshold]
+# Threshold decryption key share
+ThresholdDecryptionKeyShare = '{"DiffieHellmanPoint":"970bf092f7c542e0736c030399c7f69ffd94a9f5970647548cc6786808a5f626","EncryptedThresholdKeyShare":"3147218d037a3e5e555480a2a26599dcae60dab361b2d72b869424fed55947dbce6875360b7cdf2c3aeadc3df2ce3f903ae29049963030082b91873b62b964d721097bd75f0506e0b72b281b91cb14f39b9d885321c8e89215dc57b04f","EncryptedThresholdKeyShareHash":"722b665e73f605b6a72b99f6083018ed5ddf15a2dc5e0c4127b498a14570b239"}' #Example
```

Comment:
@pszal do we need all 3 fields here to be able to decrypt? Isn't "EncryptedThresholdKeyShare" enough?

Reply:
The hash could be omitted, but the DH point is required.

Reply:
Also, this field is currently just a filler for the docs.  It is subject to change as it is just a generic string.

Reply:
OK then let's remove the example from the doc, just put "An-encrypted-key-share" or something like that.